src/utils/invertColor.js

```diff
@@ -1,4 +1,4 @@
-export const invertColorTextArea = textarea => {
+export const invertColor = textarea => {
```

Comment:
Revert this change. This function is being used in other PRs and this will cause unnecessary merge conflicts. If need be, the change would be made by the person dealing with that PR.